pkg/local_object_storage/metabase/metadata.go

```diff
@@ -351,8 +352,8 @@ nextPrimKey:
 					continue
 				}
 			}
-			var err error
-			if dbVal, err = attrSkr.get(id, attr); err != nil {
+			dbVal, err := attrSkr.get(id, attr)
```

Comment:
Looks stylistic, not belonging to bd5a549aefaa48db7160934c83694ee0f58a69d9.

Reply:
yep, reverted